src/libraries/System.Private.CoreLib/src/System/SpanHelpers.Packed.cs

```diff
@@ -826,7 +827,6 @@ private static int ComputeFirstIndex(ref short searchSpace, ref short current, V
             return index + (int)((nuint)Unsafe.ByteOffset(ref searchSpace, ref current) / sizeof(short));
         }
 
-        [BypassReadyToRun]
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         private static int ComputeFirstIndex(ref short searchSpace, ref short current, Vector256<byte> equals)
```

Comment:
Not at the moment. I could make it so that calling an method marked with `BypassReadyToRunForIntrinsicsHelperUse` required the IsSupported checks or being called from within an appropriate `BypassReadyToRunForIntrinsicsHelperUse` method. That would make the analyzer do the "right" thing, and also make sure the helpers are called safely. Such a new feature might cause issues for the annotation on `Vector128.ShuffleUnsafe`, but possibly that's not a bad thing. That api is... odd.